src/plugins/session-timeout/session-timeout-signin-en.hbs

```diff
@@ -0,0 +1,54 @@
+	<p>The plugin is setup using the following HTML:</p>
+
+	<pre><code>&lt;span class="wb-sessto" data-wb-sessto='{"inactivity": 1200000, "reactionTime": 30000, "sessionalive": 1200000, "logouturl": "./", "signInUrl": "../signIn.html", "refreshCallbackUrl": "./"}'&gt;&lt;/span&gt;</code></pre>
+
+	<p>This allows you to configure the plugin:</p>
+	<ul>
+		<li><strong>inactivity:</strong> inactivity period of time after which the modal dialog will appear (default 20 minutes).</li>
+		<li><strong>reactionTime:</strong> period of time the user has to perform an action once the modal dialog is displayed (default 3 minutes).</li>
+		<li><strong>sessionalive:</strong> period of time for the session to stay alive until the modal dialog appears (default 20 minutes).</li>
+		<li><strong>logouturl:</strong> URL used to end the session.</li>
+		<li><strong>signInUrl:</strong> URL that users are sent to when the session has expired.</li>
+		<li><strong>refreshCallback:</strong> The function used to check the response received from refreshCallbackUrl.</li>
+		<li><strong>refreshCallbackUrl:</strong> URL used to perform an ajax request to determine the validity of the session.</li>
+		<li><strong>refreshOnClick:</strong> Determines if clicking on the document should reset the inactivity timeout and perform an ajax request (if a refreshCallbackUrl has been specified).</li>
+		<li><strong>refreshLimit:</strong> Sets the amount of time that must pass before an ajax request can be made.</li>
+		<li><strong>method:</strong> Sets the request method used for ajax requests. Recommended: GET or POST.</li>
+		<li><strong>additionalData:</strong> Additional data to send with the request.</li>
+	</ul>
+	<p>The only required parameter is logouturl, all other fields are optional. The default plugin code is:</p>
+	<pre><code>&lt;span class="wb-sessto" data-wb-sessto='{logouturl: "./"}'&gt;&lt;/span&gt;</code></pre>
+	<p><strong>Note:</strong> The inactivity, reactionTime and sessionalive parameters are set in milliseconds. For help with the time values, use this <a href="https://www.calculateme.com/Time/Minutes/ToMilliseconds.htm" rel="external">time converter</a>.</p>
+	<p><strong>Note:</strong> Your sessionalive and inactivity parameters should be equal to your web server session alive time minus the reactionTime time. If you set your sessionalive time and inactivity time to the same as your web server without taking into consideration the reactionTime time then the session will have ended by the server as soon as the popup appears to extend the session.</p>
+	<p><strong>Note:</strong> The server response needs to contain a message body. Don't use a request method (e.g. HEAD) that disallows a message body in the response.</p>
+</section>
```

Comment:
No need to add all this stuff. Simply add the current page's example's code sample. I honestly don't know why this was added in the basic example. It should all be in the documentation page, but that's not your concern.
```suggestion
```